### ISSUE TITLE
The link for replies relationships has a syntax error in the query params.

### ISSUE THREAD

**Issue by abrinckm** [CenterForOpenScience/ember-osf#147]
For example, when requesting a list of comments for a project ... 

Request:   `GET /v2/nodes/<project_id>/comments/?filter[target]=<project_id>` 

I get this response:  
```
...
"replies": {
  "links": {
     "related": {
        "href" : "https://staging-api.osf.io/v2/nodes/<project>/comments/?filter=%5Btarget%5D=ug69r64m5asn"
```

where the query params are `?filter=[target]=ug69r64m5asn` 
there is an extra '=' that should be removed. 
the link should look like this `?filter[target]=ug69r64m5asn`

I believe this is causing complications for our application when trying to retrieve comments' replies.
This may also be related to similar problems reported in #95

**Comment by abought** [comment]
This was working in a sample payload as of that previous investigation:
https://github.com/CenterForOpenScience/ember-osf/issues/95#issuecomment-231823057

However, I can confirm that the API payload for that field has changed as noted, even for the sample node linked to previously. I will ping @brianjgeiger for perspective on whether the API endpoint changed recently.

```
            "comments": {
                "links": {
                    "related": {
                        "href": "https://api.osf.io/v2/nodes/vhw6d/comments/?filter=%5Btarget%5D=vhw6d",
                        "meta": {}
                    }
                }
            },
```

**Comment by brianjgeiger** [comment]
Thanks for catching this. We'll fix it soon and let you know when it's in.

**Comment by abrinckm** [comment]
@abought whoops! I missed that detail. 
Awesome! thanks!

**Comment by brianjgeiger** [comment]
Okay, we've found the problem, and it's scheduled to go out with the next improvements release (0.101.0) which is in QA now and tentatively set for Monday. It would happen with any relationship link that had a filter.

**Comment by brianjgeiger** [comment]
This was released last night with 0.101.0 via https://github.com/CenterForOpenScience/osf.io/pull/6535
